packages/soci-snapshotter/soci-snapshotter.spec

```diff
@@ -13,6 +13,10 @@ URL: https://github.com/awslabs/soci-snapshotter
 Source0: https://github.com/awslabs/soci-snapshotter/archive/v%{gover}/soci-snapshotter-%{gover}.tar.gz
 Source1: bundled-soci-snapshotter-%{gover}.tar.gz
 Source2: bundled-cmd.tar.gz
+Source3: soci-config-toml
+# Mount for writing soci configuration
+# Generated with systemd-escape -p --suffix=mount "/etc/soci-snapshotter-grpc"
+Source100: "etc-soci\\x2dsnapshotter\\x2dgrpc.mount"
```

Comment:
Is this because of the dashes? If that's the case you can keep `etc-soci-snapshotter-grpc` as the name of the file in the sources and escape it at install time.